crates/bevy_utils/src/intern.rs

```diff
@@ -5,7 +5,7 @@
 //! and make comparisons for any type as fast as integers.
 
 use std::{
-    borrow::{Borrow, Cow},
+    borrow::Borrow,
```

Comment:
With `Interner::intern` taking a reference, we don't need to use `Borrow` anymore, since it is a no-op on references. I will push a commit fixing that after merging this PR.